common/workunit/workunit.cpp

```diff
@@ -5729,10 +5723,11 @@ void CLocalWorkUnit::setStatistic(StatisticCreatorType creatorType, const char *
         else
             statTree->removeProp("@max");
     }
-    if (creatorType==SCTsummary && scope==GLOBAL_SCOPE && kind==StTimeElapsed)
+    if (creatorType==SCTsummary && kind==StTimeElapsed && (scope==GLOBAL_SCOPE || strsame(scope, GLOBAL_SCOPE)))
```

Comment:
I put it there for speed - but I don't mind taking it out again for clarity (speed is probably irrelevant here)